FinniversKit/Sources/Assets/Icons.xcassets/AdManagement/statsClick.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "statsClick.pdf",
```

Comment:
Is this icon available as SVG? I think we should prefer SVG when possible as it is text-based. It should also have the preserve vector data flag set.